### PR TITLE
Update alluxio-env templete to contain proxy opts

### PR DESCRIPTION
**Pull request by Semptic**
The `ALLUXIO_PROXY_JAVA_OPTS` is missing from this template.

### CODE REVIEW

File: conf/alluxio-env.sh.template [hunk 1]
```diff
@@ -37,6 +37,10 @@
 # E.g. "-Dalluxio.worker.rpc.port=49999" to set worker port, "-Xms2048M -Xmx2048M" to limit the heap size of worker.
 # ALLUXIO_WORKER_JAVA_OPTS
 
+# Config properties set for Alluxio worker daemon. (Default: "")
```
Comment (gpang) [inline conf/alluxio-env.sh.template]:
```suggestion
# Config properties set for Alluxio proxy daemon. (Default: "")
```

File: conf/alluxio-env.sh.template [hunk 2]
```diff
@@ -37,6 +37,10 @@
...
+# E.g. "-Xms2048M -Xmx2048M" to limit the heap size of worker.
```
Comment (gpang) [inline conf/alluxio-env.sh.template]:
```suggestion
# E.g. "-Xms2048M -Xmx2048M" to limit the heap size of proxy.
```

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by apc999** [comment]
LGTM! Thanks for the improvement!

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Comment by haoyuan** [comment]
Thanks @Semptic !

**Pull request merged**